modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -0,0 +1,78 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
+import BigNumber from 'bignumber.js';
+import { proto } from '../../../resources/hbar/protobuf/hedera';
+import { BaseTransaction } from '../baseCoin';
+import { BaseKey } from '../baseCoin/iface';
+import { TxData } from './ifaces';
+import { toHex } from './utils';
+
+export class Transaction extends BaseTransaction {
+  private _hederaTx: proto.Transaction;
+  private _txBody: proto.TransactionBody;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  body(tx: proto.Transaction) {
+    this._txBody = proto.TransactionBody.decode(tx.bodyBytes);
+    this._hederaTx = tx;
+  }
+
+  bodyBytes(bytes: Uint8Array) {
+    this.body(proto.Transaction.decode(bytes));
+  }
+
+  /** @inheritdoc */
+  canSign(key: BaseKey): boolean {
+    return true;
+  }
+
+  /**
+   * Return the transaction in a format it can be broadcasted to the blockchain.
+   *
+   *  @returns {Uint8Array} The encoded transaction
+   */
+  toBroadcastFormat(): string {
+    return toHex(proto.Transaction.encode(this._hederaTx).finish());
```

Comment:
I think we just need to roll this into another ticket where we convert these surfaces points to use `Buffer`

Reply:
This should't be a blocker however